script/upgrades/MU01/MU01.sol

```diff
@@ -222,39 +204,46 @@ contract MU01 is IMentoUpgrade, GovernanceScript {
       console.log("Skipping BrokerProxy - already initialized");
     }
 
-    PartialReserveProxy partialReserveProxy = PartialReserveProxy(partialReserveProxyAddress);
-    if (partialReserveProxy._getImplementation() == address(0)) {
-      Reserve reserve = Reserve(contracts.deployed("Reserve"));
-      transactions.push(
-        ICeloGovernance.Transaction(
-          0,
-          address(partialReserveProxy),
-          abi.encodeWithSelector(
-            partialReserveProxy._setAndInitializeImplementation.selector,
-            reserve,
+    {
```

Comment:
did you had to do this because of a stack to deep error or is there another reason?

Reply:
Yeah stack too deep :(